src/docs/010_manual/20_install.adoc

```diff
@@ -193,7 +203,8 @@ Run the command below to tell docToolchain to download the template of your choi
 
 By now, the docToolchain wrapper `dtcw` should be in your project folder along with the arc42 template.
 
-Now Let's render it to HTML and PDF. To do so, run the commands below:
+Now Let's render arc42 to HTML and PDF. To do so, run the commands below:
+//What if I have other documents?
```

Comment:
hm. I hope the tutorial gives a good starting point and you will find out how to modify the template...?